src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -202,6 +375,57 @@ by { rw [← coe_fn_comp_to_finite_measure_eq_coe_fn,
 @[simp] lemma mass_to_finite_measure (μ : probability_measure α) :
   μ.to_finite_measure.mass = 1 := μ.coe_fn_univ
 
+variables [topological_space α]
+
+/-- The pairing of a (Borel) probability measure `μ` with a nonnegative bounded continuous
+function is obtained by (Lebesgue) integrating the (test) function against the measure. This
+is `probability_measure.test_against'`. -/
+abbreviation test_against_nn
```

Comment:
can you make it a regular definition? When a user is dealing with probability measures, you don't want the system to mention finite measures for no good reason.